HelpSource/Classes/AbstractFunction.schelp

```diff
@@ -121,7 +121,33 @@ a = Pseries(220, 220, inf).cpsmidi; a.asStream.nextN(12); // overtone series as
 method::midiratio
 method::ratiomidi
 method::ampdb
+Converts amp range (1.0 -> 0.0) to db range (ex. 0dB -> -90dB).
```

Comment:
This implies that `0.0.ampdb == -90`, which is not true, can you please find a clearer way to explain that?